ceph/rados/rados_utils_5x.py

```diff
@@ -0,0 +1,791 @@
+            "interval": f"ceph config set mgr mgr/alerts/interval {kwargs.get('interval', '5')}",
+            "smtp_from_name": f"ceph config set mgr mgr/alerts/smtp_from_name "
+            f"'{kwargs.get('smtp_from_name', 'Rados 5.0 sanity Cluster')}'",
+        }
+        try:
+            cmd = "ceph mgr module enable alerts"
+            self.node.shell([cmd])
+
+            for cmd in alert_cmds.values():
+                self.node.shell([cmd])
+
+            if kwargs.get("smtp_destination"):
+                for email in kwargs.get("smtp_destination"):
+                    cmd = f"ceph config set mgr mgr/alerts/smtp_destination {email}"
+                    self.node.shell([cmd])
+            else:
+                log.error("email addresses not provided")
+                return False
+
+        except Exception as err:
+            log.error("Error while configuring the cluster for email alerts")
+            log.error(err)
+            return False
+
+        # Printing all the configuration set for email alerts
+        cmd = "ceph config dump | grep 'mgr/alerts'"
+        log.info(self.node.shell([cmd]))
+
+        # Disabling and enabling the email alert module after setting all the config
+        try:
+            states = ["disable", "enable"]
+            for state in states:
+                cmd = f"ceph mgr module {state} alerts"
+                self.node.shell([cmd])
+                time.sleep(2)
+        except Exception as err:
+            log.error(
+                "Error while enabling/disabling alerts module after configuration"
+            )
+            log.error(err)
+            return False
+
+        # Triggering email alert
+        try:
+            cmd = "ceph alerts send"
+            self.node.shell([cmd])
+        except Exception as err:
+            log.error("Error while Sending email alerts")
+            log.error(err)
+            return False
+
+        log.info("Email alerts configured on the cluster")
+        return True
+
+    def run_rados_bench_write(self, pool_name: str, **kwargs) -> bool:
+        """
+        Method to trigger Write operations via the Rados Bench tool
+        Args:
+            pool_name: pool on which the operation will be performed
+            kwargs: Any other param that needs to passed
+
+        Returns: True -> pass, False -> fail
+
+        """
+        duration = kwargs.get("rados_write_duration", 200)
+        byte_size = kwargs.get("byte_size", 4096)
+        cmd = f"sudo rados --no-log-to-stderr -b {byte_size} -p {pool_name} bench {duration} write --no-cleanup"
+        try:
+            self.node.shell([cmd])
+            return True
+        except Exception as err:
+            log.error(f"Error running rados bench write on pool : {pool_name}")
+            log.error(err)
+            return False
+
+    def run_rados_bench_read(self, pool_name: str, **kwargs) -> bool:
```

Comment:
Likewise... we could have the method name as `bench_read`

Reply:
changed.